spring-boot-project/spring-boot-tools/spring-boot-loader/src/test/java/org/springframework/boot/loader/archive/JarFileArchiveTests.java

```diff
@@ -131,8 +132,12 @@ public void unpackedLocationsFromSameArchiveShareSameParent() throws Exception {
 	public void zip64ArchivesAreHandledGracefully() throws IOException {
 		File file = this.temporaryFolder.newFile("test.jar");
 		FileCopyUtils.copy(writeZip64Jar(), file);
-		assertThatIllegalStateException().isThrownBy(() -> new JarFileArchive(file))
-				.withMessageContaining("Zip64 archives are not supported");
+		JarFileArchive zip64Archive = new JarFileArchive(file);
+		Iterator<Entry> it = zip64Archive.iterator();
+		for (int i = 0; i < 65537; i++) {
+			assertThat(it.hasNext()).as(i + "nth file is present").isTrue();
+			it.next();
+		}
 	}
 
 	@Test
```

Comment:
This test name case should be renamed as it's not longer checking for an exception

Reply:
i renamed it